openvpn/scripts/client-connect-disconnect.sh

```diff
@@ -15,17 +15,25 @@
 # You should have received a copy of the GNU Affero General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
 
+function cleanup() {
+	if [[ $? -gt 0 ]]; then
+ 		cat </tmp/$PPID.tmp >/tmp/stderr-$PPID.log  		
```

Comment:
Would it be better to append to the file instead of overwrite it? (Same question for `2>/tmp/$PPID.tmp` redirects below.)

Reply:
Good point, do you think we could have PID collisions here?

Reply:
I honestly don't know, but my thinking is that we'd probably rather have more verbose logs and not accidentally overwrite something possibly valuable.